Assets/MixedRealityToolkit/InputSystem/Sources/BaseInputSource.cs

```diff
@@ -2,10 +2,12 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections;
+using System.Collections.Generic;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions;
 using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
 using Microsoft.MixedReality.Toolkit.Internal.Managers;
 using UnityEngine;
+using UnityEngine.XR.WSA.Input;
```

Comment:
Same here, we shouldn't be including this in base input sources.

Reply:
See above